ra-optimisation/linear-rao/pom.xml

```diff
@@ -119,5 +119,9 @@
             <artifactId>slf4j-simple</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.github.oliviercailloux</groupId>
```

Comment:
That's or-tools dependency. It used to be in the ra-optimisation-api pom but since we no longer want to depend on that package I had to move the dependency link to here.